ch_backup/logic/access.py

```diff
@@ -48,11 +79,29 @@ def restore(self, context: BackupContext) -> None:
             return
 
         has_replicated_access = self._has_replicated_access(context)
-        mark_to_rebuild = not has_replicated_access
 
-        self._restore_local(acl_ids, context, mark_to_rebuild=mark_to_rebuild)
-        if has_replicated_access:
-            self._restore_replicated(acl_ids, acl_meta, context)
+        clickhouse_access_path = context.ch_ctl_conf["access_control_path"]
+        restore_tmp_path = os.path.join(
+            context.ch_ctl_conf["tmp_path"], context.backup_meta.name
+        )
+        user = context.ch_ctl_conf["user"]
+        group = context.ch_ctl_conf["group"]
+
+        if os.path.exists(clickhouse_access_path):
+            shutil.rmtree(clickhouse_access_path)
+
+        os.makedirs(clickhouse_access_path)
+        shutil.chown(clickhouse_access_path, user, group)
```

Comment:
It makes sense to extract this lines (except related tmp path) to a separate function with `clean` parameter.